optimum/intel/openvino/quantization.py

```diff
@@ -82,21 +82,62 @@ def batch_size(self):
 
 
 class InferRequestWrapper:
-    def __init__(self, request, data_cache=None):
+    """
+    Wrapper class for OV InferRequest or CompiledModel objects that collects inputs which they were called with to
```

Comment:
We introduced .prepare_inputs() method for decoder models some time ago that can be used to collect inputs (and it is used actually). @nikita-savelyevv, can you please check if we still need this w/a with patching InferRequest?